src/v/storage/chunk_cache.cc

```diff
@@ -37,6 +39,36 @@ ss::future<> chunk_cache::start() {
       });
 }
 
+ss::future<> chunk_cache::stop() {
+    _metrics.clear();
+    return ss::now();
+}
+
+void chunk_cache::setup_metrics() {
+    if (config::shard_local_cfg().disable_metrics()) {
+        return;
+    }
+
+    namespace sm = ss::metrics;
+    _metrics.add_group(
+      prometheus_sanitize::metrics_name("chunk_cache"),
+      {
+        sm::make_gauge(
+          "total_size_bytes",
+          [this] { return _size_total; },
+          sm::description("Total size of the chunk cache in bytes")),
+        sm::make_gauge(
+          "available_size_bytes",
+          [this] { return _size_available; },
+          sm::description("Total available chunks in bytes")),
```

Comment:
```suggestion
          sm::description("Total size of all free segment appender chunks in the cache, in bytes.")),
```

Reply:
(recall all the description needs to stand alone since it appears in contexts where the group/name is not shown, so we need to repeat context)

Reply:
Can you explain?